lab_3_generate_by_ngrams/main.py

```diff
@@ -41,6 +43,27 @@ def _tokenize(self, text: str) -> Optional[tuple[str, ...]]:
         In case of corrupt input arguments, None is returned.
         In case any of methods used return None, None is returned.
         """
+        if not isinstance(text, str):
+            return None
+        self.text = text
+        text = text.lower()
+        split_text = text.split()
+        list_of_text = []
+        list_of_letters = []
+        for word in split_text:
+            word = ''.join(letter for letter in word if letter.isalpha())
```

Comment:
Why create a string from list to convert it back to list in the next line?

Reply:
Fixed